test/typedarray/delete.js

```diff
@@ -0,0 +1,36 @@
+//-------------------------------------------------------------------------------------------------------

```

Comment:
Would it be good idea to put these tests user Bugs\misc_bugs.js ?

Reply:
Also you didn't add to the rlexe.xml file

Reply:
I added the new tests under TypedArray because the delete seems to be a core spec'ed functionality for them. What would be the benefit of adding the tests under Bugs?

Reply:
I added the new tests under TypedArray because the delete seems to be a core spec'ed functionality for them. What would be the benefit of adding the tests under Bugs?

---
In reply to: [172675643](https://github.com/Microsoft/ChakraCore/pull/4786#discussion_r172675643) [](ancestors = 172675643)